### DISCUSSION TITLE
Can i change default keybinds ?

### DISCUSSION THREAD

**Discussion by sachinsenal0x64** [mikesmithgh/kitty-scrollback.nvim#90] [Q&A]
Hey title says everything thanks. :)

**Comment by mikesmithgh** [comment]
Hey @sachinsenal0x64, yes you can override the default keybinds. I have an example of this on [Advanced-Configuration](https://github.com/mikesmithgh/kitty-scrollback.nvim/wiki/Advanced-Configuration) in the section **ksb_example_keymaps_custom**. Also see [example.lua#L196](https://github.com/mikesmithgh/kitty-scrollback.nvim/blob/main/lua/kitty-scrollback/configs/example.lua#L196).

Configuring kitty-scrollback.nvim at the moment has some gotchas, I am working on improving it. This comment https://github.com/mikesmithgh/kitty-scrollback.nvim/issues/38#issuecomment-1826780094 should be useful to help you understand how you could configure it.

There is also the option `keymaps_enabled = false` if you wish to completely disable the keymaps.

Let me know if you need any help 👍
